README.md

```diff
@@ -1 +1,49 @@
-# todolist_project
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To do list - project
+
+> Creating an Awesome Books Page
+
+![screenshot](./Assets/Images/Screenshots/preview1.png)
+
+First project from Figma template
+
+
+## Built With
+
+- Html, Javascript, CSS
+- Webpack
+- Developed in VSCode
+
+
+To get a local copy up and running follow these simple example steps.
+Run:
+- `git clone https://github.com/hunter4466/todolist_project.git`
+- `cd todolist_project`
+- Open ./dist/index.html in browser
```

Comment:
- [ ] You need to add steps to run your project, there's no mention about npm install or npm start